core/src/main/java/org/geysermc/geyser/entity/EntityDefinitions.java

```diff
@@ -316,6 +316,8 @@ public final class EntityDefinitions {
                     .addTranslator(null) // Height
                     .addTranslator(null) // Glow color override
                     .build();
+
+
             TEXT_DISPLAY = EntityDefinition.inherited(TextDisplayEntity::new, displayBase)
```

Comment:
```suggestion
                    .build();
            TEXT_DISPLAY = EntityDefinition.inherited(TextDisplayEntity::new, displayBase)
```

Reply:
@stromsoftware why did you mark this as resolved? this wasn't addressed